### PR TITLE
Update uucore::utmpx

### PR DESCRIPTION
**Pull request by knight42**

### CODE REVIEW

File: tests/test_pinky.rs [hunk 1]
```diff
@@ -68,5 +58,8 @@ fn test_short_format() {
 
 #[cfg(target_os = "linux")]
 fn expected_result(args: &[&str]) -> String {
-    TestScenario::new(UTIL_NAME).cmd(UTIL_NAME).args(args).run().stdout
+    use std::process::Command;
+
+    let output = Command::new(UTIL_NAME).args(args).output().unwrap();
```
Comment (nathanross) [inline tests/test_pinky.rs]: what is the reasoning for this revert? .cmd(UTIL_NAME) has the same effect, but it also runs pinky with a cleared environment.

File: tests/test_pinky.rs [hunk 2]
```diff
@@ -68,5 +58,8 @@ fn test_short_format() {
...
+    String::from_utf8_lossy(&output.stdout).into_owned()
```
Comment (nathanross) [inline tests/test_pinky.rs]: what is the reasoning for this revert? .cmd(UTIL_NAME) has the same effect, but it also runs pinky with a cleared environment. @knight42

Comment (knight42) [inline tests/test_pinky.rs]: I expected it to run the `pinky` on the host, not the `pinky` I wrote.

Comment (nathanross) [inline tests/test_pinky.rs]: ah yeah. .ucmd() runs the uutils pinky, .cmd(UTIL_NAME) will run the pinky on the host

Comment (knight42) [inline tests/test_pinky.rs]: Oops.. my fault :cry:. I will discard my changes.

**Pull request merged**